tst/EnergyPlus/unit/Fixtures/EnergyPlusFixture.cc

```diff
@@ -337,6 +338,7 @@ namespace EnergyPlus {
 		HVACControllers::clear_state();
 		HVACDXHeatPumpSystem::clear_state();
 		HVACDXSystem::clear_state();
+		HVACFan::clearHVACFanObjects();
```

Comment:
Was there a strong reason why this wasn't just called `clear_state` like everything else?